getting_started/index.adoc

```diff
@@ -8,8 +8,33 @@
 :toc: macro
 :toc-title:
 
+[[introduction]]
+== Introduction
+{product-title} is a platform as a service (PaaS) offering from Red Hat
+that brings together Docker, Kubernetes, and an API to manage these services.
+{product-title} allows you to create and manage containers.
+Containers are standalone processes that run within their own environment.
+
+[[why-use-openshift]]
+=== Why should I use OpenShift?
+
+Containers are standalone processes that run within their own environment,
+independent of operating system and the underlying infrastructure. OpenShift
+helps you to develop, deploy, and manage container-based applications. It
+provides you with a self-service platform to create, modify, and deploy
+applications on demand. Thus enabling faster development and release life
```

Comment:
A *real* quick Google search seems to show that "thus enabling" is a phrase (or clause or whatever). 
"...deploy applications on demand, thus enabling..." Does that look right?